geometry_msgs/Twist cmd_vel
geometry_msgs/Vector3 cmd_head
geometry_msgs/Vector3 cmd_head_speed


uint8 gait_type
uint8 kick_side

uint8 CROUCH  = 0
uint8 STANDUP = 1
uint8 WENXI   = 2
uint8 KICK    = 3
uint8 GOALIE  = 4
uint8 SETUPFRONT   = 5
uint8 SETUPBACK   = 6

uint8 LEFT_KICK = 1
uint8 RIGHT_KICK = 2